lib/config/virtual_container_host.go

```diff
@@ -56,8 +56,10 @@ const (
 
 	GeneralHTTPProxy   = "HTTP_PROXY"
 	GeneralHTTPSProxy  = "HTTPS_PROXY"
+	GeneralNoProxy     = "NO_PROXY"
 	VICAdminHTTPProxy  = "VICADMIN_HTTP_PROXY"
 	VICAdminHTTPSProxy = "VICADMIN_HTTPS_PROXY"
+	VICAdminNoProxy    = "NO_PROXY"
```

Comment:
This seems inconsistent with `VICAdminHTTPProxy` and `VICAdminHTTPSProxy`. Should this be `VICADMIN_NO_PROXY`?

Reply:
NO_PROXY is used inside http.transport, so I need to put that env name
https://github.com/vmware/vic/blob/2119c415c440a2036135f423e01356be0bda52b5/lib/vicadmin/validate.go#L202